src/index.ts

```diff
@@ -183,19 +183,21 @@ const getTarEntryHandler = (
 		stream: NodeJS.ReadableStream,
 		next: (err?: Error) => void,
 	) => {
-		streamToPromise(stream).then((buffer: Buffer) => {
-			const name = normalizeTarEntry(header.name);
-			if (name === dockerfileName) {
-				const newDockerfile = transpose(buffer.toString(), opts);
-				pack.entry({ name: 'Dockerfile' }, newDockerfile);
-			} else {
-				if (name === opts.hostQemuPath && opts.qemuFileMode) {
-					header.mode = opts.qemuFileMode;
+		streamToPromise(stream)
+			.then((buffer: Buffer) => {
+				const name = normalizeTarEntry(header.name);
+				if (name === dockerfileName) {
+					const newDockerfile = transpose(buffer.toString(), opts);
+					pack.entry({ name: dockerfileName }, newDockerfile);
+				} else {
+					if (name === opts.hostQemuPath && opts.qemuFileMode) {
+						header.mode = opts.qemuFileMode;
+					}
+					pack.entry(header, buffer);
 				}
-				pack.entry(header, buffer);
-			}
-			next();
-		});
+				next();
+			})
+			.catch(next);
```

Comment:
When I added `.catch(next);` here, prettier decided that "this changes everything" :-) and reformatted the whole block. The only changes in this block are:
* Replaced `name: 'Dockerfile'` with `name: dockerfileName`
* Added `.catch(next)` because I was getting "unhandled rejection" errors in some testing.